src/lib/components/TagWidget/TagWidgetVariation1.svelte

```diff
@@ -24,7 +23,7 @@
 				<h2 class="text-6xl body-title pb-2 {titleClassNames}">{tag.title}</h2>
 				<h2 class="text-md md:text-xl">
 					<!-- eslint-disable svelte/no-at-html-tags -->
-					{@html marked(tag.content.replaceAll('<', '&lt;'))}
```

Comment:
You need to do something like this: https://github.com/be-BOP-io-SA/be-BOP/blob/5ad1de08eeedae9aea8ad4603279ecde13c2a8c2/src/lib/server/cms.ts#L183

then

to the tag's content when you fetch it from DB

In general be very careful whenever using `@html` or it can add big security holes to the project.